gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/test/java/io/gravitee/apim/core/api/use_case/ExportApiUseCaseTest.java

```diff
@@ -0,0 +1,256 @@
+
+    @InjectMocks
+    ExportApiUseCase sut;
+
+    AuditInfo auditInfo = AuditInfo.builder().organizationId(ORG_ID).build();
+
+    @BeforeEach
+    void setUp() {
+        auditService = new AuditDomainService(auditCrudService, userCrudService, new JacksonJsonDiffProcessor());
+        apiPrimaryOwnerDomainService =
+            new ApiPrimaryOwnerDomainService(
+                auditService,
+                groupQueryService,
+                membershipCrudService,
+                membershipQueryService,
+                roleQueryService,
+                userCrudService
+            );
+        apiExportDomainService =
+            new ApiExportDomainServiceImpl(
+                permissionService,
+                mediaService,
+                workflowCrudService,
+                membershipCrudService,
+                metadataCrudService,
+                pageQueryService,
+                apiCrudService,
+                apiPrimaryOwnerDomainService,
+                planCrudService
+            );
+        sut = new ExportApiUseCase(apiExportDomainService);
+        roleQueryService.initWith(
+            List.of(
+                Role
+                    .builder()
+                    .id("role-id")
+                    .scope(Role.Scope.API)
+                    .referenceType(Role.ReferenceType.ORGANIZATION)
+                    .referenceId(ORG_ID)
+                    .name("PRIMARY_OWNER")
+                    .build()
+            )
+        );
+        membershipQueryService.initWith(
+            List.of(
+                Membership
+                    .builder()
+                    .referenceType(Membership.ReferenceType.API)
+                    .referenceId(API_ID)
+                    .roleId("role-id")
+                    .memberType(Membership.Type.USER)
+                    .memberId("user id")
+                    .build()
+            )
+        );
+        userCrudService.initWith(List.of(BaseUserEntity.builder().id("user id").build()));
+        when(permissionService.hasPermission(any(), any(), any(), any())).thenReturn(true);
+    }
+
+    @AfterEach
+    void tearDown() {
+        Stream
+            .of(
+                groupQueryService,
+                membershipCrudService,
+                membershipQueryService,
+                roleQueryService,
+                userCrudService,
+                workflowCrudService,
+                metadataCrudService,
+                pageQueryService,
+                apiCrudService,
+                planCrudService
+            )
+            .forEach(InMemoryAlternative::reset);
+    }
+
+    @ParameterizedTest
+    @CsvSource(
+        delimiterString = "|",
+        textBlock = """
+        MyAPI     |  3.14.159  | myapi-3.14.159.json
+        My API    |  3.14 .159 | my-api-3.14-.159.json
+        My    API |  3.14.159  | my-api-3.14.159.json
+     """
+    )
+    void shouldExportApi(String name, String version, String expectedFilename) {
```

Comment:
It is not an explicit rule but on use case tests we use to write the test method using _. With the `DisplayNameGeneration` annotation it allows to generate a "correct" sentence in the report

I usually use _ in test method because I find it easier to read that the camelcase especially with there is composed name 🙈